benchmarks/benchmark.py

```diff
@@ -434,40 +430,52 @@ def run_benchmark():
         raise Exception("No benchmark benchmark_named {}".format(benchmark_name))
 
 
-def modify_config_props_for_mms(pargs):
-    shutil.copyfile(CONFIG_PROP_TEMPLATE, CONFIG_PROP)
-    with open(CONFIG_PROP, 'a') as f:
-        f.write('\nnumber_of_netty_threads=32')
-        f.write('\njob_queue_size=1000')
-        if pargs.gpus:
-            f.write('\nnumber_of_gpu={}'.format(pargs.gpus[0]))
+def modify_config_props_for_mms(p_args, out_dir, model_name, model_url):
```

Comment:
You should just use pargs instead of having p_args in only this one place

Reply:
p_args is a parameter which conflict with pargs. The PyCharm give warning about this conflict.

Reply:
pargs is a global, so you could remove p_args from the arguments and access the original pargs directly